app/views/dashboard/sensors/_sensor.html.haml

```diff
@@ -0,0 +1,11 @@
+= content_tag_for(:tr, sensor) do
+  %td
+    = link_to(sensor.id, show_location(sensor))
+  %td
+    %code
+      = location_names(sensor.province_id, 'province')
+  %td
+    %code
+      = sensor.sensor_rules.map(&:level).join(', ')
```

Comment:
We don't need to show levels here. We already have a link to index page of rules on the sensor show page.